Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -0,0 +1,2138 @@
+        'File': 'md5'
+    }
+
+    for ind in indicators:
+        indicator_type = tc_type_to_demisto_type.get(ind['type'], ind['type'])
+        value_field = type_to_value_field.get(ind['type'], 'summary')
+        value = ind.get(value_field, ind.get('summary', ''))
+
+        if ind.get('confidence') is not None:  # returned in specific indicator request - SDK
+            confidence = int(ind['confidence'])
+        else:
+            # returned in general indicator request - REST API
+            confidence = int(ind.get('threatAssessConfidence', 0))
+
+        if ind.get('rating') is not None:  # returned in specific indicator request - SDK
+            rating = int(ind['rating'])
+        else:
+            # returned in general indicator request - REST API
+            rating = int(ind.get('threatAssessRating', 0))
+
+        if confidence >= int(demisto.params()['rating']) and rating >= int(demisto.params()['confidence']):
+            dbot_score = 3
+            desc = ''
+            if hasattr(ind, 'description'):
+                desc = ind.description
+            mal = {
+                'Malicious': {
+                    'Vendor': 'ThreatConnect',
+                    'Description': desc,
+                }
+            }
+            if indicator_type == 'ip':
+                mal['Address'] = value
+
+            elif indicator_type == 'file':
+                mal['MD5'] = value
+                mal['SHA1'] = ind.get('sha1')
+                mal['SHA256'] = ind.get('sha256')
+
+            elif indicator_type == 'url':
+                mal['Data'] = value
+
+            elif indicator_type == 'domain':
+                mal['Name'] = value
+
+            context_path = outputPaths.get(indicator_type)
+            if context_path is not None:
+                context[context_path].append(mal)
+
+        elif int(rating) >= 1:
+            dbot_score = 2
+        else:
+            dbot_score = 1
+
+        if include_dbot_score:
+            context['DBotScore'].append({
+                'Indicator': value,
+                'Score': dbot_score,
+                'Type': indicator_type,
+                'Vendor': 'ThreatConnect'
+            })
+
+        context['TC.Indicator(val.ID && val.ID === obj.ID)'].append({
+            'ID': ind['id'],
+            'Name': value,
+            'Type': ind['type'],
+            'Owner': ind['ownerName'],
+            'Description': ind.get('description'),
+            'CreateDate': ind['dateAdded'],
+            'LastModified': ind['lastModified'],
+            'Rating': rating,
+            'Confidence': confidence,
+
+            # relevant for domain
+            'Active': ind.get('whoisActive'),
+
+            # relevant for file
+            'File.MD5': ind.get('md5'),
+            'File.SHA1': ind.get('sha1'),
+            'File.SHA256': ind.get('sha256'),
+        })
+
+        if 'group_associations' in ind:
+            if len(ind['group_associations']) >= 0:
```

Comment:
redundant, this is always true. same for the following conditions as well.
did you mean:
```suggestion
            if ind['group_associations']:
```